lib/fswatcher/fswatcher.go

```diff
@@ -31,7 +31,7 @@ type FsWatcher struct {
 	notifyModelChan chan<- FsEventsBatch
 	// All detected and to be scanned events.
 	fsEvents              FsEventsBatch
-	fsEventChan           <-chan notify.EventInfo
+	fsEventChan           chan notify.EventInfo
```

Comment:
Needed for [line 115](https://github.com/plouj/syncthing/pull/14/commits/bc0ef0a60023cf08152885aac8e33635a68fed74#diff-b32903d990e7fbffc458478f1a0db30fR115)